### PR TITLE
updates the Foundating and Community Director seats

### PR DESCRIPTION
**Pull request by ivanaszuber**

### CODE REVIEW

**Comment by jdogresorg** [comment]
FYI... I see "shall elect **four (4)** directors known as "Community Directors"".... and then later I see "The top **three (3)**candidate address with the highest number of votes will be announced as the winners of the election, and will receive the Community Director positions"

**Comment by ivanaszuber** [comment]
Thanks @jdogresorg , nice catch, fixing now :)

**Comment by robby-dermody** [comment]
**NOTE: this change to be active upon the completion of the upcoming elections**. The upcoming elections will be modified to be for four community directors, instead of three

File: Bylaws/Counterparty_Foundation_Bylaws.md [hunk 1]
```diff
@@ -145,23 +145,21 @@ All directors serving on the Board of Directors shall be selected in the manner
...
+(d) Terms and Election Dates: All directors shall hold office until their successor is elected. Except for adjustments that may be made by the Board from time-to-time to maintain or create staggered terms upon any increase or decrease in the authorized number of directors, the term of office for the Founding Director, and all Community Directors and At-Large Directors shall be one (1) year and shall run from January 1 to December 31 of the calendar year after election (with the term of the first elected Board of Directors running from April 2, 2015 until December 31, 2015). Elections of directors whose terms are expiring in any calendar year shall be held as soon as practicable following the commencement of the membership year. Unless elected by written ballot pursuant to Section 4.9, the positions of directors then expiring shall be filled and elections held at the annual meeting of members called for such purpose. The Board of Directors may issue resolutions establishing procedures governing elections of directors, provided that those resolutions are consistent with these Bylaws and the Articles.
```
Comment (FoldingCoin) [inline Bylaws/Counterparty_Foundation_Bylaws.md]: change the dates here to may 1st - April 30th

Comment (robby-dermody) [inline Bylaws/Counterparty_Foundation_Bylaws.md]: please see https://github.com/CounterpartyXCP/Documentation/pull/104 for that change. we're bringing them to the board as two separate votes

**Comment by adamkrellenstein** [comment]
ACK

**Comment by deweller** [comment]
ACK

**Comment by robby-dermody** [comment]
ACK

**Comment by FoldingCoin** [comment]
ACK

**Pull request merged**